components/katago-board.module.css

```
@@ -0,0 +1,24 @@
+  --go-board-bg: #f5f4f0;
+  --go-line: #9c9a91;
+  --go-black-fill: #18181b;
+  --go-black-edge: #3f3f46;
+  --go-white-fill: #fafafa;
+  --go-white-edge: #a1a1aa;
+  --go-mark-on-black: #fafafa;
+  --go-mark-on-white: #18181b;
+  --go-ghost-opacity: 0.4;
+}
+
+:global(html.dark) .goBoard {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)